### PR TITLE
Feature/update input wrapper

### PR DESCRIPTION
**Pull request by justinehell**
- Update InputWrapper to display additional input for updating the key (name) of the unreferenced attribute.

### CODE REVIEW

**Review by Zorin95670 — APPROVED** [review]

File: src/components/inputs/BooleanInput.vue [hunk 1]
```diff
@@ -1,13 +1,14 @@
 <template>
   <q-toggle
     v-model="localValue"
-    :label="attribute.name"
+    :label="label"
     @update:model-value="(event) => emit('update:model-value', event)"
+
```
Comment (k-y0u) [inline src/components/inputs/BooleanInput.vue:6]: empty line

File: tests/unit/components/inputs/BooleanInput.spec.js [hunk 2]
```diff
@@ -26,10 +27,27 @@ describe('Test component: BooleanInput', () => {
...
 
-    describe('Test variables: localValue', () => {
+    describe('Test variable: localValue', () => {
       it('Should match attribute.value', () => {
         expect(wrapper.vm.localValue).toEqual(true);
       });
     });
+
+    describe('Test computed: label', () => {
+      it('Should match attribute.name if attribute.definition is truthy', () => {
```
Comment (k-y0u) [inline tests/unit/components/inputs/BooleanInput.spec.js:37]: `is truthy` -> `is defined`

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: src/components/inputs/BooleanInput.vue (hunk 1 above)
Comment (justinehell) [reply src/components/inputs/BooleanInput.vue:6]: done

File: tests/unit/components/inputs/BooleanInput.spec.js (hunk 2 above)
Comment (justinehell) [reply tests/unit/components/inputs/BooleanInput.spec.js:37]: done

**Review by k-y0u — APPROVED** [review]

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**